lightningd/peer_control.c

```diff
@@ -74,13 +74,7 @@ static void destroy_peer(struct peer *peer)
 	peer_node_id_map_del(peer->ld->peers, peer);
 	if (peer->dbid)
 		peer_dbid_map_del(peer->ld->peers_by_dbid, peer);
-}
-
-static void peer_update_features(struct peer *peer,
-				 const u8 *their_features TAKES)
-{
 	tal_free(peer->their_features);
-	peer->their_features = tal_dup_talarr(peer, u8, their_features);
 }
```

Comment:
This accidentally added "tal_free(peer->their_features)" to destroy_peer.  Which is harmless, but is weird, since it's a child which is freed anyway.

Reply:
oops yeah was trying to figure out memory issue and forgot to remove